.circleci/config.yml

```diff
@@ -916,6 +916,44 @@ jobs:
             - store_artifacts:
                   path: ~/transformers/reports
 
+    run_tests_tokenization_clip:
+        working_directory: ~/transformers
+        docker:
+            - image: circleci/python:3.7
+        environment:
+            OMP_NUM_THREADS: 1
+            TRANSFORMERS_IS_CI: yes
+        resource_class: large
+        parallelism: 1
+        steps:
+            - checkout
+            - restore_cache:
+                  keys:
+                      - v0.4-torch-{{ checksum "setup.py" }}
+                      - v0.4-{{ checksum "setup.py" }}
+            - run: pip install --upgrade pip
+            - run: pip install .[all,testing]
+            - save_cache:
+                  key: v0.4-onnx-{{ checksum "setup.py" }}
+                  paths:
+                      - '~/.cache/pip'
+            - run: pip install ftfy
+            - save_cache:
+                  key: v0.4-torch-{{ checksum "setup.py" }}
+                  paths:
+                      - '~/.cache/pip'
+            - run: python utils/tests_fetcher.py | tee test_preparation.txt
```

Comment:
I think testing this in the nightly is enough, so we can remove the test fetcher and move the job to the nightlies only.

Reply:
Noted, I moved it to the list of jobs to run at night in [this commit](https://github.com/huggingface/transformers/pull/15067/commits/a0fc90a8fd90586d087bd2efd40eb4744bfa109a). I've also removed the `save_cache` items so don't hesitate to tell me if I was wrong to do it.